### PR TITLE
DO-52-Fixed Update Member issue

### PR DESCRIPTION
**Pull request by pepcus-jitendra**
DO-52-Fixed Update Member issue for inserting new address entity or social network entity

### CODE REVIEW

File: src/main/java/com/pepcus/directoryonline/api/validator/BetweenLimitedType.java [hunk 1]
```diff
@@ -9,27 +9,27 @@
  */
 public class BetweenLimitedType extends ValidationType {
 
-  private double lowerLimited;
+  private Integer lowerLimited;
```
Comment (sandeep-pepcus) [inline src/main/java/com/pepcus/directoryonline/api/validator/BetweenLimitedType.java:12]: why we convert double to integer?

File: src/main/java/com/pepcus/directoryonline/api/validator/DataValidator.java [hunk 2]
```diff
@@ -145,7 +145,7 @@ private static void betweenZeroAndOne(String propName, Number value, Application
     between(propName, value, 0, 1, error);
   }
 
-  public static final ValidationType between(double lower, double upper) {
+  public static final ValidationType between(Integer lower, Integer upper) {
```
Comment (sandeep-pepcus) [inline src/main/java/com/pepcus/directoryonline/api/validator/DataValidator.java:148]: why we convert double to integer at validation framework. This is common layer. If we want for integer then should create override method.

Comment (pepcus-jitendra) [reply src/main/java/com/pepcus/directoryonline/api/validator/DataValidator.java]: @sandeep-pepcus Changes has been done.

File: src/main/java/com/pepcus/directoryonline/api/validator/BetweenLimitedType.java (hunk 1 above)
Comment (pepcus-jitendra) [reply src/main/java/com/pepcus/directoryonline/api/validator/BetweenLimitedType.java]: @sandeep-pepcus Changes has been done.

**Pull request merged**